### PR TITLE
Task #6156 - Share social network

### PR DESCRIPTION
**Pull request by taylq**
https://edu-redmine.framgia.vn/issues/6156

### CODE REVIEW

**Comment by taylq** [comment]
readay

**Comment by taylq** [comment]
ready

File: app/controllers/rater_controller.rb [hunk 1]
```diff
@@ -1,13 +1,11 @@
 class RaterController < ApplicationController
 
   def create
+    @book = Book.find_by id: params[:id]
```
Comment (huuphuc93) [inline app/controllers/rater_controller.rb]: check đk k tìm thấy @book

File: app/views/books/show.html.erb [hunk 2]
```diff
@@ -10,6 +10,13 @@
           <div class="col-sm-9 style-padding">
             <div class="row">
               <h1 class="book-name"><%= @book.name %></h1>
+              <div class="star2" data-id="<%= @book.id %>">
+                <%= rating_for @book, "like", disable_after_rate: false %>
+              </div>
+              <div class="star1">
+                <%= render "star", book: @book, layout: false %>
+              </div>
+              <%= social_share_button_tag(@book.name) %>
```
Comment (huuphuc93) [inline app/views/books/show.html.erb]: xóa  ()

**Comment by huuphuc93** [comment]
commented

**Comment by taylq** [comment]
ready

**Pull request merged**